news/issues/issue2.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: single
+title:  "*Flying*  Issue 2"
+sidebar:
+  nav: news
+redirect_from: /news/latest/
+---
+### Laying the groundwork for Spinnaker success
```

Comment:
All these headings are level three, but there are no level-2 headings. I recommend promoting these to h2.